src/server/json_family.cc

```diff
@@ -390,8 +391,9 @@ void SendJsonValue(ConnectionContext* cntx, const JsonType& j) {
 }
 
 OpResult<string> OpGet(const OpArgs& op_args, string_view key,
-                       vector<pair<string_view, JsonExpression>> expressions, bool should_format,
-                       const OptString& indent, const OptString& new_line, const OptString& space) {
+                       vector<pair<string_view, optional<JsonExpression>>> expressions,
```

Comment:
I do not remember, expressions are passed by value by design?
if you are changing already, can you please rename to OpJsonGet?

Reply:
I just did touch that part, I refactored some of the bulky boilerplate parts. Changed, renamed (but all others don't have json 🤷🏻‍♂️)